.github/scripts/set-dependency-versions.js

```diff
@@ -3,12 +3,12 @@ const fs = require('fs');
 const PACKAGE_JSON_PATH = './package.json';
 
 const DEPENDENCY_VERSIONS = {
-    'apify': process.env.APIFY_VERSION,
-    'puppeteer': process.env.PUPPETEER_VERSION,
-    'playwright': process.env.PLAYWRIGHT_VERSION || '1.7.1',
-    'playwright-chromium': process.env.PLAYWRIGHT_VERSION || '1.7.1',
-    'playwright-firefox': process.env.PLAYWRIGHT_VERSION || '1.7.1',
-    'playwright-webkit': process.env.PLAYWRIGHT_VERSION || '1.7.1',
+    'apify': process.env.APIFY_VERSION || 'beta',
+    'puppeteer': process.env.PUPPETEER_VERSION || 'latest',
```

Comment:
I think the defaults were left out for a reason here. Most likely to prevent building something by accident.